src/screens/FinancialSummary/FinancialSummaries.tsx

```diff
@@ -0,0 +1,151 @@
+import React from 'react';
+import BaseHeader from '../../components/BaseComponents/BaseHeader';
+import { TextField, Typography, Button } from '@material-ui/core';
+import OutlinedCardList from '../../components/OutlinedCardList';
+import { RouteComponentProps } from 'react-router-dom';
+import SearchIcon from '@material-ui/icons/Search';
+import { connect } from 'react-redux';
+import { RootState } from '../../lib/redux/store';
+import { withStyles, createStyles, Theme } from '@material-ui/core/styles';
+import { FinancialSummaryRecord } from '../../lib/airtable/interface';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      flexGrow: 1,
+      height: '100vh',
+    },
+    content: {
+      textAlign: 'left',
+      color: theme.palette.text.secondary,
+      padding: '0px 20px 80px',
+      width: '100%',
+      height: '80vh',
+      overflow: 'auto',
+    },
+    payButton: {
+      borderRadius: '12px',
+      height: '30px',
+      width: '80px',
+      color: 'white',
+      backgroundColor: theme.palette.primary.main,
+    },
+    viewButton: {
+      borderRadius: '12px',
+      height: '30px',
+      width: '80px',
+      color: theme.palette.primary.main,
+      backgroundColor: 'white',
+      border: `1px solid ${theme.palette.primary.main}`,
+    },
+    searchContainer: {
+      float: 'right',
+      padding: '0px 30px',
+    },
+    searchIcon: {
+      fontSize: '20px',
+      color: theme.palette.text.primary,
+      marginTop: '-5px',
+    },
+    search: {
+      border: `1px solid ${theme.palette.primary.main}`,
+      borderRadius: '10px',
+      width: '130px',
+      height: '25px',
+      padding: '2px 10px',
+    },
+    title: {
+      marginTop: '20px',
+    },
+  });
+
+interface FinancialSummariesProps extends RouteComponentProps {
+  classes: {
+    root: string;
+    content: string;
+    payButton: string;
+    viewButton: string;
+    searchIcon: string;
+    searchContainer: string;
+    search: string;
+    title: string;
+  };
+  financialSummaries?: any;
+  location: any;
+}
+
+function FinancialSummaries(props: FinancialSummariesProps) {
+  const { classes, financialSummaries } = props;
+  let paidFinancialSummaries: any[] = [];
+  let unpaidFinancialSummaries: any[] = [];
+
+  const formatFinancialSummaries = (summaries: FinancialSummaryRecord[]) => {
+      summaries.forEach(summary => {
+        const entry = { number: summary.totalAmountBilled, label: summary.lastUpdated, unit: 'MMK'};
```

Comment:
I think according to the Figma we want the large text to show period rather than totalAmountBilled. For the period, let's just use the month and year? I think there's a field in financial summaries that already has the period as a string, we might be able to just show that

![image](https://user-images.githubusercontent.com/35501399/108930353-a4ace480-75fa-11eb-97c6-2badae004d66.png)